paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/PaymentMethodCommonDataModel.kt

```diff
@@ -1,26 +1,27 @@
 package com.stripe.android.paymentsheet.forms
 
+
+data class VisualFieldLayout(val visualFieldLayout: List<Section>) {
+    val allFields get() = visualFieldLayout.map { it.field }
+}
```

Comment:
What is the significance of `Visual` here? I would imagine anything to do with a layout would be visual.